Containerfile.rag-content

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/ubi9/python-311 as lightspeed-rag-builder
+
+USER 0
+RUN dnf install -y --nodocs --setopt=keepcache=0 --setopt=tsflags=nodocs git ruby gem
+RUN pip install torch --index-url https://download.pytorch.org/whl/cpu
+RUN pip install PyYAML huggingface_hub llama_index langchain_community \
+                faiss-cpu llama-index-vector-stores-faiss sentence_transformers \
+                llama-index-embeddings-huggingface
+RUN gem install asciidoctor
+
+WORKDIR /workdir
+RUN git clone --single-branch --branch enterprise-4.14 https://github.com/openshift/openshift-docs.git
```

Comment:
we've been using 4.14 because that's what we had in the right format.  But if you're taking the existing docs repo content and converting it into a format that we can generate embeddings from, then we should be starting from the 4.15 docs.